src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
I`d recommend such naming.
```suggestion
    public static final Map<String, Integer> FRUITS = new HashMap<>();
```